VSTPlugin.cpp

```diff
@@ -213,9 +213,11 @@ void VSTPlugin::closeEditor()
 			effect->dispatcher(effect, effEditClose, 0, 0, nullptr, 0);
 		}
 
-		editorWidget->close();
-		editorWidget->deleteLater();
-		editorWidget = nullptr;
+		EditorWidget *temp = editorWidget;
+		editorWidget       = NULL;
```

Comment:
Just a very minor nitpick, but remember to use `nullptr` with cpp source files rather than `NULL`. If you could change that I'd be grateful.

Reply:
fixed